src/panel/poi/PoiPanel.jsx

```diff
@@ -249,6 +249,12 @@ export default class PoiPanel extends React.Component {
       </div>
       {(!isMobile || panelSize === 'maximized') && <div className="poi_panel__fullContent">
         <PoiBlockContainer poi={poi} covid19Enabled={covid19Enabled} />
+        {isFromPagesJaunes(poi) &&
+          <div className="poi_panel__info-partnership">
+            {_('In partnership with ')}
+            <img src="./statics/images/pj.png" alt="Pages Jaunes" width="80" height="16" />
```

Comment:
It seems vertically aligned on baseline whereas it should be centered.

Reply:
it might be easier with an `inline-flex`